ethstats/state.go

```diff
@@ -214,6 +214,28 @@ func (s *State) WriteNodeStats(nodeId string, stats *NodeStats) error {
 	return nil
 }
 
+func (s *State) DeleteOlderData(days int) error {
+	tx, err := s.db.Beginx()
+	if err != nil {
+		return err
+	}
+
+	query := fmt.Sprintf("DELETE FROM public.blocks WHERE public.blocks.created_at < now() - interval '%d days';", days)
```

Comment:
I would prefer that instead of using `now` we directly compute in Go the day to remove stuff. It gets more clearer of which data are we removing.

Reply:
IMO I would remove the cron logic from the PR, just maintain the DELETE logic so we can call the deletion from the Cloud provider side.

Reply:
I'll then let it remain untouched for @vcastellm .